dex-it/src/test/scala/com/wavesplatform/it/WsSuiteBase.scala

```diff
@@ -56,6 +56,7 @@ trait WsSuiteBase extends MatcherSuiteBase with HasWebSockets with DiffMatcherWi
           .partitionMap {
             case x: WsError => x.asLeft
             case x: WsPingOrPong => x.asRight
+            case _ => throw new IllegalArgumentException(s"Unexpected error")
```

Comment:
`s` is not required here, only when we substitute an expression. So `"s` could be replaced with `"` :)

Reply:
fixed